client/cz-cardview.html

```diff
@@ -167,7 +171,9 @@
         if (views.length > this.columns)
           views.splice(this.columns, views.length - this.columns);
         this.set('views', views);
+
         this.customStyle['--column-percent'] = (100/this.columns) + '%';
+        this.customStyle['--total-percent'] = ((this.columns * 100)/(Number(this.columns) + 2)) + '%';
```

Comment:
`Number()` should probably be around both references to `this.columns` or neither.